<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_Broadcast</class>
 <widget class="QWidget" name="GUI_Broadcast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>5</number>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lab_status">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>3 users listening</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_error">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cannot Broadcast</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_retry">
       <property name="text">
        <string>Retry</string>
       </property>
       <property name="icon">
        <iconset resource="../../Resources/Icons.qrc">
         <normaloff>:/Icons/undo.svg.png</normaloff>:/Icons/undo.svg.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_dismiss">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>Dismiss</string>
       </property>
       <property name="icon">
        <iconset resource="../../Resources/Icons.qrc">
         <normaloff>:/Icons/delete.svg.png</normaloff>:/Icons/delete.svg.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_dismiss_all">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>Dismiss all</string>
       </property>
       <property name="icon">
        <iconset resource="../../Resources/Icons.qrc">
         <normaloff>:/Icons/delete.svg.png</normaloff>:/Icons/delete.svg.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="combo_clients">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Resources/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
